Panic when press UP on empty list

There is an panic when list is empty and change function is set up on key press UP:

```go
package main

import "github.com/rivo/tview"

func main() {
	app := tview.NewApplication()
	list := tview.NewList()
	list.SetBorder(true).SetTitle("Panic when press UP on empty list")
	list.SetChangedFunc(func(index int, mainText, secondaryText string, shortcut rune) {
		// empty
	})
	if err := app.SetRoot(list, true).Run(); err != nil {
		panic(err)
	}
}
```

panic: runtime error: index out of range [-1] [recovered]
	panic: runtime error: index out of range [-1]

goroutine 1 [running]:
github.com/rivo/tview.(*Application).Run.func1(0xc0000d8180)
	/Users/bronos/Projects/go/src/github.com/BronOS/pkg/mod/github.com/rivo/tview@v0.0.0-20200204110323-ae3d8cac5e4b/application.go:160 +0x82
panic(0x1189ae0, 0xc0003ec000)
	/usr/local/go/src/runtime/panic.go:679 +0x1b2
github.com/rivo/tview.(*List).InputHandler.func1(0xc0003e8000, 0xc0003ea010)
	/Users/bronos/Projects/go/src/github.com/BronOS/pkg/mod/github.com/rivo/tview@v0.0.0-20200204110323-ae3d8cac5e4b/list.go:547 +0x378
github.com/rivo/tview.(*Box).WrapInputHandler.func1(0xc0003e8000, 0xc0003ea010)
	/Users/bronos/Projects/go/src/github.com/BronOS/pkg/mod/github.com/rivo/tview@v0.0.0-20200204110323-ae3d8cac5e4b/box.go:160 +0x5d
github.com/rivo/tview.(*Application).Run(0xc0000d8180, 0x0, 0x0)
	/Users/bronos/Projects/go/src/github.com/BronOS/pkg/mod/github.com/rivo/tview@v0.0.0-20200204110323-ae3d8cac5e4b/application.go:245 +0x3c0
main.main()
	/Users/bronos/Projects/go/src/github.com/BronOS/tview_test/main.go:12 +0x21b

--- a/list.go
+++ b/list.go
@@ -474,6 +474,15 @@ func (l *List) Draw(screen tcell.Screen) {
 // InputHandler returns the handler for this primitive.
 func (l *List) InputHandler() func(event *tcell.EventKey, setFocus func(p Primitive)) {
 	return l.WrapInputHandler(func(event *tcell.EventKey, setFocus func(p Primitive)) {
+		if event.Key() == tcell.KeyEscape {
+			if l.done != nil {
+				l.done()
+			}
+			return
+		} else if len(l.items) == 0 {
+			return
+		}
+
 		previousItem := l.currentItem
 
 		switch key := event.Key(); key {
@@ -498,10 +507,6 @@ func (l *List) InputHandler() func(event *tcell.EventKey, setFocus func(p Primit
 				if l.selected != nil {
 					l.selected(l.currentItem, item.MainText, item.SecondaryText, item.Shortcut)
 				}
-			}
-		case tcell.KeyEscape:
-			if l.done != nil {
-				l.done()
 			}
 		case tcell.KeyRune:
 			ch := event.Rune()

